ship_compliant.gemspec

```
@@ -21,7 +21,7 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
 
   s.add_dependency "activesupport", ['> 5.0', '<7']
-  s.add_dependency "savon", "~> 2.3"
+  s.add_dependency "savon"
```

```suggestion
  s.add_dependency "savon", "< 3"
```

Re: Nick's comment. I think this would be a reasonably safe thing to do in this gemspec.